common/network_manager_test.go

```diff
@@ -209,6 +209,12 @@ func TestOnRequestPausedBlockedIPs(t *testing.T) {
 	}
 }
 
+type MetricInterceptorMock struct{}
+
+func (m *MetricInterceptorMock) urlTagName(ctx context.Context, urlTag string) (string, bool) {
+	return "", false
+}
```

Comment:
It is not the best win, but this would be a one-liner if we had used a function value instead of an interface as well.

Reply:
I think i'll leave it as it for now, thanks for the info though.